core/products/perpetual.go

```diff
@@ -454,6 +464,37 @@ func (p *Perpetual) calculateFundingPayment(internalTWAP, externalTWAP *num.Uint
 	return fundingPayment, &fundingRate
 }
 
+func (p *Perpetual) calculateInterestTerm(externalTWAP, internalTWAP *num.Uint, delta int64) *num.Int {
+	// a year in nanoseconds, used to get delta in terms of years
+	year, _ := num.DecimalFromString("31536000000000000")
```

Comment:
could make that a globak var so it's not done every time,
could also be made like this so it look less like out of nowhere I guess:
```
var year = num.DecimalFromInt((24 * 365 * time.Hour).Nanoseconds())
```

Reply:
Done.